src/app/pages/signing/signing.component.css

```diff
@@ -135,6 +140,47 @@ markdown {
   border-radius: 0.4em;
 }
 
+/* || Embed Photo */
+
+#disclaimer-button {
+  color: white;
+}
+
+#embed-image-holder {
+  max-width: 100%;
+}
+
+#embed-image-card {
+  border: 0.0625em solid grey;
+  min-height: calc(1.5em + .75rem + 2px);
```

Comment:
Why the weird mixture of units? I thought that `rem` isn't supposed to be combined with `em`.